modules/ROOT/pages/7.3-release-notes.adoc

```diff
@@ -29,17 +29,23 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
 The following premium plugin updates were released alongside {productname} {release-version}.
 
-=== <Premium plugin name 1>
+=== Enhanced Code Editor
 
-The {productname} {release-version} release includes an accompanying release of the **<Premium plugin name 1>** premium plugin.
+The {productname} {release-version} release includes an accompanying release of the **Enhanced Code Editor** premium plugin.
 
-**<Premium plugin name 1>** includes the following <fixes, changes, improvements>.
+**Enhanced Code Editor** includes the following improvements.
 
-==== <Premium plugin name 1 change 1>
+==== By default, code is now formatted automatically when the Enhanced Code Editor view is opened.
 
-// CCFR here.
+Previously, the Enhanced Code Editor plugin did not automatically format code with indentation, resulting in less readable source code for users.
+
+With {productname} {release-version}, the new `advcode_prettify_editor` option has been introduced. By default, this option is set to `true`, ensuring that HTML content is automatically formatted upon opening the Enhanced Code Editor view.
+
+Additionally, users can manually format pasted HTML code in the Enhanced Code Editor with a single click using the new "Format Code" button.
```

Comment:
Using the button user can both format all Enhanced Code Editor content or selected section.

Reply:
```suggestion
Additionally, users can now format either all content within the Enhanced Code Editor or a specific context selection when clicking on the "Format Code" button.
```